src/utils/dashboardUtilFunctions.js

```diff
@@ -410,7 +410,7 @@ export function transformInitialDataForSunburst(data, level1 = 'program', level2
     color: level1Colors[parseInt(index, 10)],
     caseSize: level1Child.caseSize,
     children: level1Child[level1Children].map((level2Child, index2) => ({
-      title: `${level1Child.program} : ${level2Child[level2]}`,
```

Comment:
@skiran86  Can you add a fallback if level1 child is not defined?

Reply:
@AjayDoddapaneni There is a level1 = 'program value for level1 in the params which we usually get from the place where it is called to generate the sunburst data, in case the user didn't pass anything for level1 then there is already a default value for it is 'program' which will work as a fallback for this so it will never break